src/pages/index.module.css

```diff
@@ -1,23 +1,405 @@
+		margin-top: 6rem;
+	}
+	.companion .provider:nth-of-type(6) {
+		margin-top: 3rem;
+	}
+}
+
+/* ----- Frameworks ------  */
+.section-stack {
+	position: relative;
+	padding: 8rem 0;
+	background-image: url('/img/grid-8.svg');
+	background-position: center;
+	background-repeat: repeat-y;
+	background-size: 100%;
+}
+.section-stack::after {
+	content: '';
+	position: absolute;
+	top: 0;
+	left: 0;
+	display: block;
+	width: 100%;
+	height: 100%;
+	background: radial-gradient(
+		circle,
+		rgba(255, 255, 255, 0) 0%,
+		rgba(255, 255, 255, 1) 75%
+	);
+}
+
+.section-stack > div {
+	position: inherit;
+	z-index: 10;
+}
+.section-stack h2,
+.section-stack p {
+	background-color: white;
+}
+
+.frameworks {
+	display: grid;
+	grid-template-columns: repeat(4, minmax(0, 1fr));
+	margin: 2rem 0;
+}
+@media screen and (min-width: 50rem) {
+	.section-stack {
+		background-image: url('/img/grid-12.svg');
+	}
+	.frameworks-wrapper {
+		display: grid;
+		grid-template-columns: repeat(12, minmax(0, 1fr));
+		align-items: start;
+	}
+	.frameworks-wrapper div:first-of-type {
+		grid-column: 1 / 6;
+	}
+	.frameworks-wrapper div:nth-of-type(2) {
+		grid-column: 7 / 13;
+		box-shadow: 0 1px 1px rgba(0, 0, 0, 0.12), 0 2px 2px rgba(0, 0, 0, 0.12),
+			0 4px 4px rgba(0, 0, 0, 0.12), 0 8px 8px rgba(0, 0, 0, 0.12),
+			0 16px 16px rgba(0, 0, 0, 0.12);
+	}
+}
+
+.framework-input {
+	position: absolute;
+	clip: rect(0, 0, 0, 0);
+}
+.framework-input + label svg {
+	width: 2rem;
+}
+.framework-input + label {
+	display: flex;
+	padding: 1rem;
+	border: 2px solid lightgray;
+	background-color: white;
+	justify-content: center;
+	flex-direction: column;
 	align-items: center;
+	font-weight: 600;
+	font-family: var(--ifm-font-family-monospace);
+	font-size: 0.85rem;
+	cursor: pointer;
+}
+.framework-input:not(:first-of-type) + label {
+	border-left: none;
+}
+.framework-input + label:first-of-type {
+	border-radius: 5px 0px 0px 5px;
+}
+.framework-input + label:last-of-type {
+	border-radius: 0px 5px 5px 0px;
+}
+.framework-input:checked + label {
+	border: 2px solid var(--blue);
+}
+.framework-input:not(:first-child):checked + label {
+	margin-left: -2px;
+}
+.framework-input:not(:checked) + label {
+	color: darkgray;
+}
+.framework-input:not(:checked) + label:hover {
+	color: gray;
+}
+.framework-input:checked + label span {
+	color: black;
+}
+.framework-input[value='React']:checked + label {
+	color: #61dafb;
+}
+.framework-input[value='Vue']:checked + label {
+	color: #4fc08d;
+}
+.framework-input[value='Svelte']:checked + label {
+	color: #ff3e00;
+}
+.framework-input[value='Angular']:checked + label {
+	color: #dd0031;
+}
+
+/* And much more */
+
+.section-much-more ul {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+@media screen and (min-width: 50rem) {
+	.section-much-more ul {
+		display: grid;
+		grid-template-columns: repeat(2, minmax(0, 1fr));
+		gap: 0 1rem;
+		align-items: start;
+	}
+	.section-much-more li {
+		margin: 0;
```

Comment:
It looks more readable without the margin, let’s remove?

Reply:
I think you mean _with_? Either way instead of margin I defined `gap` for the same effect. Does it need to be bigger?